include/fiction/algorithms/simulation/sidb/operational_domain.hpp

```diff
@@ -455,9 +464,16 @@ class operational_domain_impl
                                          step_point{current_contour_point.x - 1, current_contour_point.y};
 
         auto current_neighborhood = moore_neighborhood(current_contour_point);
-        auto next_point           = current_contour_point == backtrack_point ?
-                                        current_neighborhood.front() :
-                                        next_clockwise_point(current_neighborhood, backtrack_point);
+
+        step_point front_neighborhood{};
```

Comment:
What is a `front_neighborhood`? What is it used for?

Reply:
sorry, this is superflous. Thank you!